packages/docs/docs/getting-started/faqs.md

```diff
@@ -0,0 +1,28 @@
+---
+
+title: FAQs
+---
+
+# ❓FAQs
+
+What is Unirep?
+
+* UniRep is an anonymous reputation protocol. The protocol allows developers to build applications that associate data with anonymous users. 
+
+What is an epoch key?
+
+* Epoch Keys are how a user stays anonymous while receiving data. An epoch key is a temporary anonymous identity or persona that a user uses to interact with a Unirep application across epochs that are renewed each epoch.
+
+What is the architecture of a Unirep application?
+
+* A Unirep application will have the architecture of web app <-> relay <-> blockchain. 
```

Comment:
I think relay is optional
but you can note that if there is no relay, your ethereum address will be exposed
and I think you can note that the web app (client) is used by users and relay is hosted by attester
It is quite complicated so I think you can use a diagram to illustrate it 😅

Reply:
<img width="1009" alt="" src="https://github.com/Unirep/Unirep/assets/17507085/b334b766-15be-47c2-b22e-82003fd321d2">

Reply:
maybe @ChialiT can design this as well 🙏🏻